org.d3cod3.Mosaic.yml

```diff
@@ -0,0 +1,449 @@
+app-id: org.d3cod3.Mosaic
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: mosaic
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
```

Comment:
since you don't have `--socket=wayland` you shouldn't have `--socket=fallback-x11`

Reply:
I thought you needed it for xwayland. I'll correct it

Reply:
nothing specific is needed for Xwayland. It is designed as an Xserver, the client has no idea it's Xwayland vs Xorg.

Reply:
The fallback-x11 is only meant as "if wayland isn't there, allow X11"